core/src/main/java/edu/wpi/grip/core/GRIPCoreModule.java

```diff
@@ -29,6 +29,28 @@
     private final EventBus eventBus = new EventBus(this::onSubscriberException);
 
     public GRIPCoreModule() {
+        //Set up the global level logger. This handles IO for all loggers.
+        Logger globalLogger = LogManager.getLogManager().getLogger("");//This is our global logger
```

Comment:
Not sure if this helps here, but Guice has built in bindings for `Logger`

https://github.com/google/guice/wiki/BuiltInBindings